docs/cheatsheet.rst

```diff
@@ -100,15 +100,32 @@ Function Visibility Specifiers
-- ``anonymous`` for events: Does not store event signature as topic.
-- ``indexed`` for event parameters: Stores the parameter as topic.
-- ``virtual`` for functions and modifiers: Allows the function's or modifier's
-  behaviour to be changed in derived contracts.
-- ``override``: States that this function, modifier or public state variable changes
-  the behaviour of a function or modifier in a base contract.
++------------+-------------------------------------+--------------------------------------------+
+| Use                  | Description                                            | Modifier      |
++============+=====================================+============================================+
+| For functions        | Disallows modification or access of state              | ``pure``      |
++------------+-------------------------------------+--------------------------------------------+
+| For functions        | Disallows modification of state                        | ``view``      |
++------------+-------------------------------------+--------------------------------------------+
+| For functions        | Allows them to receive Ether together with a call      | ``payable``   |
++------------+-------------------------------------+--------------------------------------------+
+| For state variables  | Disallows assignment (except initialisation), does not | ``contant``   |
+|                      | occupy storage slot                                    |               |
++------------+-------------------------------------+--------------------------------------------+
+| For state variables  | Allows exactly one assignment at construction time and | ``immutable`` |
+|                      | is constant afterwards. Is stored in code              |               |
++------------+-------------------------------------+--------------------------------------------+
+| For events           | Does not store event signature as topic                | ``anonymous`` |
++------------+-------------------------------------+--------------------------------------------+
+| For event parameters | Stores the parameter as topic                          | ``indexed``   |
++------------+-------------------------------------+--------------------------------------------+
+| For functions and    | Allows the function’s or modifier’s behaviour to be    | ``virtual``   |
+| modifiers            | changed in derived contracts                           |               |
++------------+-------------------------------------+--------------------------------------------+
+| Function, modifier   | Function, modifier or public state variable changes the| ``override``  |
+| and state variable   | behaviour of a function or modifier in a base contract |               |
++------------+-------------------------------------+--------------------------------------------+
```

Comment:
The table syntax is wrong, and doesn't actually render a table. In general, before opening a PR or pushing any changes, try to run/build them; in this case, you can generate the docs by running `docs/docs.sh` and then looking at the generated build files in `docs/_build` directory. This particular HTML will be generated in 
```
docs/_build/html/cheatsheet.html
```
If you need a reference as to what a properly formed table looks like - take a look at
```
docs/types/operator-precedence-table.rst
```